imgui.go

```diff
@@ -686,3 +686,62 @@ func IsItemHovered() bool {
 func IsKeyPressed(key int) bool {
 	return C.iggIsKeyPressed(C.int(key)) != 0
 }
+
+// BeginColumns calls BeginColumnsV().
+func BeginColumns(count int, label string) {
+	BeginColumnsV(count, label, ColumnsFlagsNone)
+}
+
+// BeginColumnsV creates a column layout of the specified number of columns.
+func BeginColumnsV(count int, label string, flags int32) {
```

Comment:
This also applies to the terse variant `BeginColumns()`:
You confused me here because the public name here is "BeginColumns", yet the actual function used is `Columns`, while the identically named `BeginColumns` of ImGui is an internal function - and is not called.

Please rename this one to `Columns` and `ColumnsV`.